src/analysis/normed_space/multilinear.lean

```diff
@@ -379,6 +374,30 @@ normed_group.of_core _ ⟨op_norm_zero_iff, op_norm_add_le, op_norm_neg⟩
 instance to_normed_space : normed_space 𝕜' (continuous_multilinear_map 𝕜 E G) :=
 ⟨λ c f, f.op_norm_smul_le c⟩
 
+lemma op_norm_prod (f : continuous_multilinear_map 𝕜 E G) (g : continuous_multilinear_map 𝕜 E G') :
+  ∥f.prod g∥ = ∥(f, g)∥ :=
```

Comment:
Is there a reason why you write `∥(f, g)∥` instead of `max (∥f∥) (∥g∥)` on the right? The latter would seem a little bit more straightforward to me, as there are many norms on the direct product and the reader might not be aware that the default one is the sup one in mathlib.